data/src/main/java/com/microsoft/azure/kusto/data/format/CslBoolFormat.java

```diff
@@ -0,0 +1,28 @@
+package com.microsoft.azure.kusto.data.format;
+
+import com.microsoft.azure.kusto.data.Ensure;
+
+public class CslBoolFormat extends CslFormat {
+    private final Boolean value;
+
+    public CslBoolFormat(boolean value) {
+        this.value = value;
+    }
+
+    @Override
+    public String getType() {
+        return "bool";
+    }
+
+    @Override
+    public Boolean getValue() {
+        return value;
+    }
+
+    @Override
+    String getValueAsString() {
+        Ensure.argIsNotNull(value, "value");
+
+        return Boolean.TRUE.equals(value) ? "true" : "false";

```

Comment:
Why not simply `return value ? "true" : "false";`

Reply:
Because `Boolean` can be null, in which case testing it will throw a NPE.